agent/softwarecontainer-agent.xml

```diff
@@ -25,6 +25,16 @@
             <arg direction="out" type="u" name="pid" />
         </method>
 
+        <method name="FreezeContainer">
+            <arg direction="in" type="u" name="containerID" />
+            <arg direction="out" type="b" name="success" />
+        </method>
+
+        <method name="ThawContainer">
```

Comment:
I think we can have better names for "freeze" and "thaw" that are less tied to LXC. For instance "pause" and "unpause". The point is  that we might want to consider what wording we use before pushing it into our API.